pom.xml

```diff
@@ -9,9 +9,42 @@
         <version>9</version>
     </parent>
 
-    <groupId>uk.co.qualitycode</groupId>
+    <groupId>io.github.excitedkettle</groupId>
```

Comment:
revert this before merging, I just left it in to show the changes I had to make to deploy it.